README.md

```diff
@@ -12,9 +12,9 @@
 The original buffers are kept intact and copies are only done as necessary. Any reads that require the use of a single original buffer will return a slice of that buffer only (which references the same memory as the original buffer). Reads that span buffers perform concatenation as required and return the results transparently.
 
 ```js
-const BufferList = require('bl')
+const { BufferList } = require('bl')
```

Comment:
I can live with it but I'm not entirely comfortable making the old style completely disappear from the README. This is only 4 more characters but it's not clear from the README that the old style is still possible. Thoughts from anyone else?

Reply:
Worth mentioning it _is_ [documented later that `BufferListStream` is the default export](https://github.com/rvagg/bl/pull/74/files#diff-04c6e90faac2675aa89e2176d2eec7d8R225-R231) (so you can continue to use it as before) and the way we've done this won't break existing users...unless there's some unforseen issue!

It's also worth pointing out that `BufferListStream` is the equivalent of what users currently get when they `require('bl')` and that the documentation has changed to effectively recommend using the non-streamy version (first example is using `BufferList` not `BufferListStream`).

I half assumed this would be a major release but it's completely backwards compatible so it doesn't have to be. We could tone down the change in the README to not _look_ so "major" if you were planning on releasing as a minor and it is a concern.

IMHO as a major release this documentation change is absolutely fine, as a minor it's probably still fine, but it has the _potential_ to confuse some existing users (although not actually break their code).